core/scss/components/lockup/_lockup.scss

```diff
@@ -0,0 +1,155 @@
+//
+// Markup: ../templates/components/lockup/lockup.twig
+//
+// Style guide: Components.Lockup
+//
+.su-lockup {
+  @include display-grid;
+  @include grid-columns(1fr);
+  @include grid-rows(auto auto auto auto);
+  align-items: end;
+  width: fit-content;
+  text-decoration: none;
+
+  @media #{$breakpoint-md} {
+    @include grid-columns(auto auto 1fr);
+    @include grid-rows(auto auto);
+  }
+}
+
+.su-lockup__wordmark {
+  @include logo;
+  display: block;
+  -ms-grid-row: 1;
+  -ms-grid-row-span: 1;
+  grid-row: 1 / span 1;
+  -ms-grid-column: 1;
+  -ms-grid-column-span: 1;
+  grid-column: 1 / span 1;
+  color: $color-cardinal-red;
+  font-size: 36px;
+  line-height: 1;
+
+  @media #{$breakpoint-md} {
+    font-size: 46px;
+    height: 33px;
+    line-height: .96;
+  }
+
+  // Hide pipe/horizontal line on smaller devices if there is no line 5
+  + .su-lockup__pipe {
+    display: none;
+
+    @media #{$breakpoint-md} {
+      display: block;
+    }
+  }
+}
+
+//Put the pipe in its own CSS grid cell so that its height auto adjusts to the tallest part of the lockup
+.su-lockup__pipe {
+  @include margin(7px null);
+  display: block;
+  -ms-grid-row: 3;
+  -ms-grid-row-span: 1;
+  grid-row: 3 / span 1;
+  -ms-grid-column: 1;
+  -ms-grid-column-span: 1;
+  grid-column: 1 / span 1;
+  content: "";
+  width: 118px;
+  height: 1px;
+  background-color: $color-black;
+
+  @media #{$breakpoint-md} {
+    @include margin(0 7px);
+    grid-row: 1 / span 1;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 1;
+    grid-column: 2 / span 1;
+    -ms-grid-column: 2;
+    -ms-grid-column-span: 1;
+    display: inline-block;
+    width: 1px;
+    height: 100%;
+  }
+
+}
+
+.su-lockup__sitename-line1,
+.su-lockup__sitename-line2,
+.su-lockup__sitename-line5 {
+  @include font-smoothing;
+  display: block;
+  grid-column: 1 / span 1;
+  -ms-grid-column: 1;
+  -ms-grid-column-span: 1;
+  color: $color-black;
+}
+
+
+.su-lockup__sitename-line1 {
+  grid-row: 4 / span 1;
+  -ms-grid-row: 4;
+  -ms-grid-row-span: 1;
+  font-size: 22px;
+  line-height: .8;
+
+  @media #{$breakpoint-md} {
+    grid-row: 1 / span 1;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 1;
+    grid-column: 3 / span 1;
+    -ms-grid-column: 3;
+    -ms-grid-column-span: 1;
+    font-size: 30px;
+  }
+}
+
+.su-lockup__sitename-line2 {
+  display: none;
+}
+
+.su-lockup__sitename-line5 {
+  grid-row: 2 / span 1;
+  -ms-grid-row: 2;
+  -ms-grid-row-span: 1;
+  font-size: 20px;
+  line-height: 1.1;
+
+  @media #{$breakpoint-md} {
+    @include padding(8px null null);
+    grid-row: 2 / span 1;
+    -ms-grid-row: 2;
+    -ms-grid-row-span: 1;
+    grid-column: 1 / span 3;
+    -ms-grid-column: 1;
+    -ms-grid-column-span: 3;
+    font-size: 27px;
+  }
+}
+
+.su-lockup__sitename-line1--uppercase,
+.su-lockup__sitename-line5--uppercase {
+  text-transform: uppercase;
+  font-weight: 700;
+}
+
+.su-lockup__sitename-line5--uppercase {
+  font-size: 18px;
```

Comment:
I'm surprised by the extreme difference in font size between line 5's default (27px on desktop) and the uppercase variant (18px on desktop). But in looking at the [Stanford Open Framework site lockup guide](https://docs.google.com/document/d/14_92mZHRocYWwFQfMWdTz5kQpT1QoSGj4AazAsRM5mc/edit?usp=sharing), I see why you did that. So I'm good with this now, but we may revisit later.